library/everystudent/src/main/java/org/cru/godtools/everystudent/EveryStudent.java

```diff
@@ -126,6 +126,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        mTracker.setActiveActivity(this);
```

Comment:
good catch here, it was an oversight since this was legacy code that wasn't part of the current common framework